core/tauri/src/app.rs

```diff
@@ -1580,6 +1583,10 @@ fn on_event_loop_event<R: Runtime, F: FnMut(&AppHandle<R>, RunEvent) + 'static>(
     manager.on_window_close(label);
   }
 
+  if let RuntimeRunEvent::OpenURLs(urls) = &event {
+    app_handle.trigger_global("open-urls", Some(to_string(urls).unwrap()));
```

Comment:
This _serialize_ / _deserialize_ looks weird but I don't know what's the best way